tests/test_bench.py

```
@@ -14,54 +14,66 @@ def setup_function(function):
+    def setup():
+        reset_graph()
+    benchmark.pedantic(run_query, args=(f"UNWIND range(1, {n}) AS x CREATE (:N {{id: x}})-[:R]->(:N {{id: x + 1}})",), setup=setup, rounds=5, warmup_rounds=1)
 
 @pytest.mark.parametrize("n", [
     1, 10, 100, 1000, 10000, 100000, 1000000
 ])
 def test_match_node(benchmark, n):
-    query(f"UNWIND range(1, {n}) AS x CREATE (:N {{id: x}})")
-    benchmark(query, f"MATCH (n:N) RETURN n")
+    run_query(f"UNWIND range(1, {n}) AS x CREATE (:N {{id: x}})")
+    benchmark(run_query, f"MATCH (n:N) RETURN n")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)